texdoc/__init__.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+
+"""texdoc extension
+
+This is an extension to search for LaTeX documentation.
+
+Synopsis: <trigger> <query>"""
+
+import os
+import re
+import subprocess
+
+from shutil import which
+from albertv0 import *
+
+__iid__          = "PythonInterface/v0.3"
+__prettyname__   = "TeXdoc"
+__version__      = "1.0"
+__trigger__      = "td"
+__author__       = "Florian Adamsky"
+__dependencies__ = ["texdoc"]
+
+iconPath   = os.path.dirname(__file__)+"/texdoc-logo.svg"
+texdoc_cmd = ["texdoc", "-I", "-q", "-s", "-M"]
+
+if which("texdoc") is None:
+    raise Exception("'texdoc' is not in $PATH.")
+
+def handleQuery(query):
+    if not query.isTriggered:
+        return
+
+    query.disableSort()
+
+    stripped_query = query.string.strip()
+
+    if stripped_query:
+        process = subprocess.run(["texdoc", "-I", "-q", "-s", "-M", stripped_query],
```

Comment:
probably you want to use the `textdoc_cmd` that you've defined earlier

Reply:
```suggestion
        process = subprocess.run(texdoc_cmd + [stripped_query],
```

Reply:
Done! Thx!